contrib/demo-windows.md

```diff
@@ -0,0 +1,85 @@
+## Requirement
+Here is the quick instruction for anyone who wants to check out the remote execution POC (proof of concept) in the super agent.
+
+You don't need to have knowledge of the techonology behind super agent. But you need to have basic knowledge of Microsoft Remote Desktop, Powershell command in Windows and know how to use a browser.
+
+## Connect to windows server and set up Chrome browser
+
+Use Microsoft Remote Desktop to connect to the windows server.  If you are using Mac, you need to install the Remote Desktop here: https://itunes.apple.com/us/app/microsoft-remote-desktop/id715768417?mt=12
+
+The demo only works with Chrome or Firefox. Install Chrome browser <https://www.google.com/chrome/browser/desktop/>
+
+## Install Luvi and Lit
+
+Download the Powershell script here and run it: <https://github.com/luvit/lit/blob/master/get-lit.ps1>
+
+## Install Luvit and Fife
+Now that we have lit installed, we can use it to automatically download and build any published app.
+
+Let's install fife, the super-agent.
+```
+Set-ExecutionPolicy Unrestricted
+.\lit.exe make lit://virgo-agent-toolkit/fife fife.exe
+```
+
+## Configure Fife
+We'll configure fife for local-only access to keep things simple and secure.
+
+Create `fife.conf` in the current directory with the following content
+
+```
+mode = "standalone"
+ip = "127.0.0.1"
+port = 7000
+webroot = "./super-agent"
+```
+## Download the client
+
+If you don't have git, download and install git from here <https://git-scm.com/download/win>
+
+Clone the super-agent repo recursivly from the the same directory where fife.exe file was created in the above command
+
+```
+git clone --recursive https://github.com/virgo-agent-toolkit/super-agent.git
+```
```

Comment:
hmm, that submodule sure is causing us trouble.  It's only needed for syntax highlighting in the IDE and the text editor in the fancy desktop demo.  I wonder if the app will work without it.  If so, then people can just download the zip from github for the super-agent repo and not need to install git.